### PR TITLE
Updating ReadMe for the ACE Website Open Source Project

### PR DESCRIPTION
**Pull request by josuevicente33**
Updated the ReadMe to include more details on how to run the code for it's intended audience of students. Update on prerequisites required. Added detail on how to set  .env variables. Changed ports numbers to match.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,22 +74,32 @@ Create a personal branch
...
 
 Deploying Project to gh-pages
 * npm run deploy
 * commit/push to your branch
 * merge to main branch
 * result will show up on gh-pages in roughly 5-10 minutes
 
+## Common Issue
+* If you are unable to get the website running after using `npm install` under `/client`, you might have to use:
+    * `npm instaill --force`, then try using script again
```
Comment (charlieforward9) [inline README.md:100]: install*

**Review by charlieforward9 — CHANGES_REQUESTED** [review]
Great additions to the ReadMe. Just noticed a small typo, apart from that, good to go on my end.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+
+# Prerequites
+Before beginning the installation process, you must make sure to have Node Version Manager for Windows. The website requires
+node version 16 or 18 in order to run. NVM for windows can be installed here: https://github.com/coreybutler/nvm-windows
+
+To install:
+    * Use the latest installer and once that is done, in your terminal use
+    * nvm
+        * This will give you the version of NVM if installed properly
+    * nodee -v
```
Comment (jaARke) [inline README.md:22]: I think this should just be `node -v`

**Review by jaARke — CHANGES_REQUESTED** [review]
Found one more typo.

Also, trying using all the commands in \` ` characters so it's clear that's what they are.

Other than that, looks great.

**Pull request merged**